lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2683,11 +2684,16 @@ static void finishPropertyWrapperImplInfo(VarDecl *var,
     }
   }
 
-  if (wrapperSetterIsUsable)
+  if (wrapperSetterIsUsable) {
```

Comment:
You might want to clean this up with an early return too ("if (!wrapperSetterIsUsable) { info = StorageImplInfo::getImmutableComputed(); return;")